### ISSUE TITLE
[CKEditor 5] v7.21.0 crashes after opening a MathType dialog

### ISSUE THREAD

**Issue by Mgsy** [wiris/html-integrations#51]
## Steps to reproduce

1.  Install the latest (v7.21.0) version of MathType for CKEditor 5.
2.  Click on the MathType/ChemType icon in the toolbar.

## Current result

The editor crashes.

## Error

```
Uncaught TypeError: Cannot read property 'length' of undefined
    at Function.get (stringmanager.js:20)
    at new ModalDialog (modal.js:114)
    at Core.openModalDialog (core.src.js:702)
    at CKEditor5Integration.openNewFormulaEditor (integrationmodel.js:287)
    at CKEditor5Integration.openNewFormulaEditor (integration.js:127)
    at MathTypeCommand.openEditor (commands.js:51)
    at MathTypeCommand.execute (commands.js:29)
    at MathTypeCommand.<anonymous> (observablemixin.js:255)
    at MathTypeCommand.fire (emittermixin.js:209)
    at MathTypeCommand.<computed> [as execute] (observablemixin.js:259)
```

## Notes

The issue is in StringManager. It seems that this line causes it:

https://github.com/wiris/html-integrations/blob/stable/packages/mathtype-html-integration-devkit/src/stringmanager.js#L17

It should verify if `language` is not `undefined`.

It seems that the issue has been introduced earlier, as even in v7.20.0 opening the MathType dialog gives a warning:

```javascript
Unknown language undefined set in StringManager.
```

It hasn't crashed because there was no operations on `this.language`. In v7.21.0 the following code has been added and it throws, as `this.language` is `undefined`:

```javascript
// Cut down on strings. e.g. en_US -> en
if (language.length > 2) {
  language = language.slice(0, 2);
}
```

**Comment by xaviripo** [comment]
Noticed! We will add a fix for this in the next version.

**Comment by xaviripo** [comment]
This has been fixed in the [7.22.0 version](https://www.npmjs.com/package/@wiris/mathtype-ckeditor5/v/7.22.0).
